src/bot/events/update-guilds.ts

```diff
@@ -74,12 +76,17 @@ export class UpdatedGuilds {
       });
 
     await this.discord_helper.findOrCreateBiggestVirginRole(guild_ent);
-    await this.discord_helper.findOrCreateVirginityBotChannel(guild_ent);
+    const channel = await this.discord_helper.findOrCreateVirginityBotChannel(
+      guild_ent,
+    );
```

Comment:
I think that we might have an issue around channel creation. We should look through the logs to check that this actually works or not.